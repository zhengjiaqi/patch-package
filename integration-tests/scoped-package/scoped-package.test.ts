import { runIntegrationTest } from "../runIntegrationTest"
runIntegrationTest({
  projectName: "scoped-package",
  shouldProduceSnapshots: true,
})
