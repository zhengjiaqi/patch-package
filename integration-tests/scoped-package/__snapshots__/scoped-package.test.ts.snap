// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test scoped-package: add.d.ts should contain patch-package 1`] = `
"SNAPSHOT: add.d.ts should contain patch-package
import { patch-package } from \\"./index\\";
export = patch-package;
END SNAPSHOT"
`;

exports[`Test scoped-package: left-pad typings should contain patch-package 1`] = `
"SNAPSHOT: left-pad typings should contain patch-package
// Definitions by: Zlatko patch-package <https://github.com/Goldsmith42/>
END SNAPSHOT"
`;
